integration_tests/sdk/aqueduct_tests/param_test.py

```diff
@@ -296,10 +296,11 @@ def __init__(self):
     @op
     def must_be_picklable(input):
         """
-        Unable to check that the input is pickleabe, since `pickle.loads()`
+        Unable to check that the input is picklable, since `pickle.loads()`
         complains about `import of module 'param_test' failed`.
         """
-        assert input == EmptyClass
```

Comment:
asserts inside a packaged function doesn't work well when the server is on a different machine, since it may not have `pytest` installed.